src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,49 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Message } from '../enums/Message';
 
 export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setError(false);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
   return (
     <>
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
           <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
+            {!!people.length && <PeopleFilters />}
           </div>
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
+              {isLoading && <Loader />}
 
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+              {error && <p data-cy="peopleLoadingError">{Message.Error}</p>}
 
-              <p data-cy="noPeopleMessage">There are no people on the server</p>
+              {!isLoading && !error && !people.length && (
+                <p data-cy="noPeopleMessage">{Message.NoPeople}</p>
+              )}
 
-              <p>There are no people matching the current search criteria</p>
+              {false && <p>{Message.Missmatch}</p>}
```

Comment:
you should use a condition instead of false because right now this message will never appear